src/app/index.ts

```diff
@@ -68,6 +75,12 @@ module.exports = yo.extend({
       desc: 'Use the prerelease version of the project template.'
     });
 
+    this.option('is-test', {
```

Comment:
I said this before but continer "--testing" (or "--for-testing" or "--test") for the name. You don't really need "is" since specifying a flag indicates true by virtue of it being present (and false if not specified).

Reply:
changed to --test